docs/tutorials/Snow/data_tutorial.jl

```diff
@@ -0,0 +1,166 @@
+# # Scraping SNOTEL Data
+
+# This tutorial shows you how to make use of the code developed
+# for scraping SNOTEL site data in order to generate datasets for use 
+# in training artificial intelligence models for seasonal snow forecasting.
+# The code below contains a commented version of the code used to produce
+# `cleandata.csv`, which is used in the base tutorial for snow forecasting,
+# as well as the paper. However, exploration of the optional arguments
+# or requesting of alternative [SNOTEL data codes](https://www.nrcs.usda.gov/wps/portal/wcc/home/dataAccessHelp/webService/webServiceReference#elementCodes) offers
+# additional utility in creating alternative data sets for further investigation.
+
+# We begin by importing the data tools module, as well as other required packages:
+using ClimaLSM.Snow.DataTools
+using ClimaLSM
+using DataFrames, CSV
+
+# We then define constants that will be used in the cleaning of the SNOTEL data,
+# such as conversion constants from imperial to metric units, and the sensor limits
+# defined in the [SNOTEL Engineering Handbook](https://directives.sc.egov.usda.gov/OpenNonWebContent.aspx?content=27630.wba). Some SNOTEL sensors measure
+# in imperial units, and some measure in metric units, and the data portal will round
+# converted values if a sensor stream is requested in units other than its original
+# measurement. Therefore, we will scrape data in the originally measured units to limit
+# systemic errors.
+
+const inch2meter = 0.0254
+const kmphr2mps = 5.0 / 18.0
+
+filter_val = Dict{Symbol, Tuple{Real, Real}}(
+    :SWE => (0.0, 250.0),
+    :z => (0.0, 420.0),
+    :precip => (0.0, 250.0),
+    :rel_hum_avg => (10.0, 100.0),
+    :sol_rad_avg => (0.0, 1500.0),
+    :wind_speed_avg => (0.0, 216.0),
+    :air_temp_avg => (-55.0, 60.0),
+)
+
+scales = Dict{Symbol, Real}(
+    :SWE => inch2meter,
+    :z => inch2meter,
+    :precip => inch2meter,
+    :rel_hum_avg => 0.01,
+    :wind_speed_avg => kmphr2mps,
+)
+
+# We next proceed to outline which stations will be scraped by defining
+# a dictionary of station IDs, paired with the date range to be scraped.
+# "`start`" refers to 1850-01-01 or the first available date, while "`end`"
+# refers to 2023-01-01 or the last available date. Most of these stations
+# are commented out for the sake of speed and readability in generating
+# the tutorial, but can be uncommented to yield the full dataset found in
+# `cleandata.csv` used in the [base tutorial](@ref base_tutorial.jl). Stations were
+# selected based upon their availability of the features utilized in
+# creating the model used in the paper.
+
+good_stations = Dict{Int, Tuple{String, String}}(
+    1030 => ("start", "2016-01-01"),
+    1053 => ("2010-01-01", "end"),
+    #1083 => ("2013-01-01", "end"),
```

Comment:
if you like, this:
```
#=
code 
code
code
=#
```
can comment out a block of code without requiring # on each line